go/ct/rlz/conditions.go

```diff
@@ -724,7 +734,20 @@ func (c *isAddressCold) Restrict(generator *gen.StateGenerator) {
 }
 
 func (c *isAddressCold) GetTestValues() []TestValue {
-	return IsAddressWarm(c.key).GetTestValues()
+	property := Property(c.String())
+	domain := boolDomain{}
+	restrict := func(generator *gen.StateGenerator, isWarm bool) {
+		key := c.key.GetVariable()
+		c.key.BindTo(generator)
+		if isWarm {
+			generator.BindToWarmAddress(key)
+		} else {
+			generator.BindToColdAddress(key)
+		}
+	}
+	return []TestValue{
+		NewTestValue(property, domain, false, restrict),
+	}
```

Comment:
This code is mostly a duplicate of `isAddressWarm`'s `GetTestValue`. Please factor it out into a common utility function.